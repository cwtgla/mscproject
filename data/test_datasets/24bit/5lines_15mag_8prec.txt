32767.441
7091.237
0.996
-1500.104565
-32767.2859