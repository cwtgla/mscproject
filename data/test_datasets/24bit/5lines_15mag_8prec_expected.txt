127
255
44
27
179
23
0
0
99
133
220
10
255
255
28